wiki/FitNesseRoot/HsacExamples/SlimTests/UtilityFixtures/FileFixture.wiki

```diff
@@ -14,57 +14,59 @@ Filenames may be manipulated.
-|check                  |text in     |greeting.txt                |Hello John                      |
-|set value              |John        |for                         |name                            |
-|show                   |create      |greeting.html               |using     |greeting.ftl.html    |
-|show                   |text in     |greeting.html                                                |
-|show                   |content of  |greeting.html                                                |
-|note                   |filename's containing either / or \ will point to sub-directories         |
-|show                   |copy        |${DIR}/greeting.txt         |to        |copy/greetingCopy.txt|
-|check                  |text in     |${DIR}\copy\greetingCopy.txt|Hello John                      |
-|check                  |filename of |${DIR}\copy\greetingCopy.txt|greetingCopy.txt                |
-|check                  |base name of|${DIR}\copy\greetingCopy.txt|greetingCopy                    |
-|check                  |extension of|${DIR}\copy\greetingCopy.txt|txt                             |
-|note                   |you can create a new file by "appending" one line into it                 |
-|set directory          |http://files/test                                                         |
-|$appended=             |append      |first                       |to        |appended.txt         |
-|check                  |text in     |$appended                   |first                           |
-|note                   |you can append content to an existing file                                |
-|append                 |!- line-!   |to                          |$appended                       |
-|check                  |text in     |$appended                   |first line                      |
-|note                   |you can append content as a new line to an existing file                  |
-|append                 |second line |to                          |$appended |on new line          |
-|convert line endings of|$appended   |to unix                                                      |
-|check                  |text in     |$appended                                                    |!-first line
+|script                 |file fixture                                                                    |
+|note                   |default directory used by the fixture is inside the files section               |
+|$def_dir=              |get directory                                                                   |
+|$created=              |create        |bye1.txt                      |containing |Bye Bye 1             |
+|repeat at most         |50            |times                                                            |
+|                       |Do not forget to set 'repeat at most times', or else the loop may run endlessly!|
```

Comment:
this is the only addition here.

Reply:
That was indeed not that easy to spot :-)